argocd/aad-pod-identity/Chart.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: "v2"
 name: "aad-pod-identity"
 version: "0"
+description: |
+  This chart enables Kubernetes applications to access cloud resources securely with Azure Active Directory (AAD).
```

Comment:
@mcanevet is this accurate for you?

Reply:
The project description is `Assign Azure Active Directory Identities to Kubernetes applications.`. Maybe we should set that here.

Reply:
fixed